core/domain/taskqueue_services_test.py

```diff
@@ -59,6 +59,32 @@ def test_exception_raised_when_email_task_params_is_not_serializable(self):
                 params,
                 0)
 
+    def test_defer_makes_the_correct_request(self):
+        correct_fn_identifier = '/task/deferredtaskshandler'
+        correct_url = feconf.TASK_URL_DEFERRED
+        correct_queue_name = taskqueue_services.QUEUE_NAME_EMAILS
+        correct_payload = {
+            'fn_identifier': '/task/deferredtaskshandler',
+            'args': [],
+            'kwargs': {}
+        }
+
+        swap_create_http_task = self.swap_with_checks(
```

Comment:
Our convention is to put `swap` at the end, e.g. `create_http_task_swap`

Reply:
Modified name according to the convention